### PR TITLE
made login occur on new page instead of in pop up

### PR DESCRIPTION
**Pull request by ameletyan**

### CODE REVIEW

File: src/util/login.js [hunk 1]
```diff
@@ -1,29 +1,52 @@
...
+		var credential = error.credential;
+		// ...
+	});
 
-        // Pass uid to the specified callback.
-        uidCallback(user.uid);
-    }).catch(function(error) {
-        console.log(error);
-    });
+	// POP UP
```
Comment (sperry94) [inline src/util/login.js:27]: commented lines should be deleted

File: src/util/login.js [hunk 2]
```diff
@@ -1,29 +1,52 @@
...
-                });
-            }
-        });
+	// NEW PAGE
+	firebase.auth().signInWithRedirect(provider);
+	firebase.auth().getRedirectResult().then(function(result) {
+		if (result.credential) {
+			// This gives you a Google Access Token. You can use it to access the Google API.
+			var token = result.credential.accessToken;
+			// ...
```
Comment (sperry94) [inline src/util/login.js:12]: need to add the user to the users ref if they aren't there already

**Review by sperry94 — CHANGES_REQUESTED** [review]
commented lines should be deleted and the user needs to be added to the users ref if they are new users

**Comment by calebhsu** [comment]
@ameletyan 
Is this still relevant or can we close this PR?

**Pull request closed without merging**